_posts/2019-11-07-Kubernetes-native_Infrastructure-Managed_Baremetal_with_Kubernetes_Operators_and_OpenStack_Ironic.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Kubernetes-native Infrastructure: Managed Baremetal with Kubernetes Operators and OpenStack Ironic - Steve Hardy, Red Hat"
+date: 2019-11-07T13:02:00+02:00
+draft: false
+categories: ["kubernetes", "metal3", "operator", "baremetal", "openstack", "ironic"]
+author: Pedro Ibáñez Requena
+---
+
+## Conference talk: Open Infraestructure Days UK 2019; Kubernetes-native Infrastructure: Managed Baremetal with Kubernetes Operators and OpenStack Ironic - Steve Hardy, Red Hat
+
+In this session you could hear about a new effort to enable baremetal Kubernetes deployments using native interfaces, and in particular the Kubernetes Operator framework, combined with OpenStack Ironic.
+
+This approach aims to seamlessly integrate your infrastructure with your workloads, including baremetal servers, storage and container/VM workloads.  All this can be achieved using kubernetes native applications, combined with existing, proven deployment and storage tooling.
+
+I were covered the options around Kubernetes deployments today, the specific approach taken by the new Kubernetes-native "Metalkube" project, and the status/roadmap of this new community effort.
+
+
+## Speakers
+[Steve Hardy](https://twitter.com/hogepodge) is Senior Principal Software Engineer at Red Hat, currently involved in kubernetes/OpenShift deployment and architecture. He is also an active member of the OpenStack community, and has been a project team lead of both the Heat (orchestration) and TripleO (deployment) projects.
```

Comment:
This twitter handle is wrong, you could refer to my blog at http://hardysteven.blogspot.com - I don't really do twitter

Reply:
Sure, changed.